sostene12/my-brand-api
# PR: merge-to-main

#### What does this PR do?
-implement merge to main
#### Description of Task to be completed
-merge the develop(default ) branch in other to deploy on cyclic
#### How should this be manually tested?
-clone the repo using git clone repo's URL
-change branch git checkout develop
-create a .env file and put the required environment variables from .env.example file
-make the request methods with the postman and also check the unit test with npm run test
#### What are the relevant pivotal tracker stories?
-PR:merge develop-to main
